### DISCUSSION TITLE
Changing boundary condition type in pyFluent

### DISCUSSION THREAD

**Discussion by RandSim** [ansys/pyfluent#3483] [Q&A]
Hi,
I am trying to change the boundary condition type of the zone 'inlet_test-1' but I am having no luck. See code below for more information. Could you point me in the right direction?
Alex,

PS: I am looking for a pyFluent answer, if none is given on this thread, one can use this TUI command instead: solver.tui.define.boundary_conditions.modify_zones.zone_type("inlet_test-1","mass-flow-inlet")

**Error message:**
"
>>> solver.setup.boundary_conditions.set_zone_type.argument_names
['zone_list', 'new_type']
>>> solver.setup.boundary_conditions.set_zone_type(['inlet_test-1','velocity_inlet'])
Traceback (most recent call last):
  File "<console>", line 1, in <module>
TypeError: Command.__call__() takes 1 positional argument but 2 were given
"

**Comment by mkundu1** [comment]
Please try
```
solver.settings.setup.boundary_conditions.set_zone_type(zone_list=["inlet_test-1"], new_type="mass-flow-inlet")
```

**Comment by RandSim** [comment]
Thanks for the timely response Mainak, this worked!
